js/packages/phoenix-client/src/experiments/instrument.ts

```diff
@@ -0,0 +1,37 @@
+import { diag, DiagConsoleLogger, DiagLogLevel } from "@opentelemetry/api";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-proto";
+import { resourceFromAttributes } from "@opentelemetry/resources";
+import { SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { SEMRESATTRS_PROJECT_NAME } from "@arizeai/openinference-semantic-conventions";
+import { HeadersOptions } from "openapi-fetch";
+
+export function instrument({
```

Comment:
I would maybe re-name this as register and return the provider so you can un-register